src/pages/index.js

```diff
@@ -27,8 +27,8 @@ const featureRows = [
       link: 'docs/develop/sdk/quickstart',
       description: (
         <>
-          Start building with the KILT Protocol. Issue and verify your first credential
-          or setup an app.
+          Start building with KILT. Issue and verify your first credential
+          or set up an app.
```

Comment:
New line missing? This is automatically formatted right?
```suggestion
          Start building with KILT.
          Issue and verify your first credential or set up an app.
```

Reply:
Yes, there's nothing we can do about it without configuring some weird lint rules, I think.